src/ImageSharp/Common/Helpers/Guard.cs

```diff
@@ -14,7 +14,7 @@ namespace ImageSharp
     /// Provides methods to protect against invalid parameters.
     /// </summary>
     [DebuggerStepThrough]
-    internal static class Guard
+    public static class Guard
```

Comment:
After seeing so many pieces of shared (duplicated) code and the comment from @antonfirsov and now making something public that should not be public I do think we should use `InternalsVisibleTo` for the Helpers and keep them all together inside the ImageSharp project.